src/flag.rs

```diff
@@ -82,10 +85,17 @@
 //!
 //! fn main() -> Result<(), Error> {
 //!     let got = Arc::new(AtomicBool::new(false));
+//! #   #[cfg(not(windows))]
 //!     signal_hook::flag::register(signal_hook::SIGUSR1, Arc::clone(&got))?;
+//! #   #[cfg(windows)]
+//! #   signal_hook::flag::register(signal_hook::SIGTERM, Arc::clone(&got))?;
 //!     unsafe {
+//! #       #[cfg(not(windows))]
 //!         let pid = libc::getpid();
+//! #       #[cfg(not(windows))]
 //!         libc::kill(pid, signal_hook::SIGUSR1);
+//! #       #[cfg(windows)]
+//! #       libc::raise(signal_hook::SIGTERM);
 //!     }
```

Comment:
Let's simplify this one :-). Let's use SIGTERM and raise in both windows and linux versions.

Reply:
Isn't it good to choose a "user-defined" signal? I chose `SIGTERM` here because `SIGTERM` is said not to be generated under Windows, and unfortunately there's no `SIGUSR*` in Windows. https://docs.microsoft.com/ja-jp/cpp/c-runtime-library/reference/signal?view=vs-2019

> The **SIGILL** and **SIGTERM** signals are not generated under Windows.